language_service/src/completion.rs

```diff
@@ -471,7 +480,15 @@ impl CompletionListBuilder {
                                                 Some(start) => {
                                                     additional_edits.push(TextEdit {
                                                         new_text: format!(
-                                                            "open {};{indent}",
+                                                            "open {}{};{indent}",
```

Comment:
Is it doable to have the package alias just be part of the stored namespace for items? I ask because prefixing with the package name should really happen everywhere the namespace appears, right? I can see from trying it out that the alias does _not_ appear when hovering over items from external code, which seems erroneous. It would be good if we could solve all these issues centrally by making the package alias part of the stored namespace; otherwise we'll need to do this prefixing logic a lot, right?

Reply:
I think it's worth filing a follow up issue for this and doing a deeper investigation. To your point, Scott, there are other places in language service like hover that don't have the alias. I think we'd want to double check stuff like type errors as well (I think those don't use fully qualified names at all, but it's worth a double check, because if they do and alias is missing it could cause confusion due to seeming collisions). It's possible this is already well plumbed for those type of compiler error messages, but either way it's worth combing through and making sure we are consistent throughout.